src/main/java/io/vertx/httpproxy/interceptors/impl/RemovePrefixPathInterceptorImpl.java

```diff
@@ -11,10 +11,11 @@
 
 package io.vertx.httpproxy.interceptors.impl;
 
-import io.vertx.httpproxy.ProxyInterceptor;
 
-public class RemovePrefixPathInterceptorImpl implements ProxyInterceptor {
+public class RemovePrefixPathInterceptorImpl extends PathInterceptorImpl {
```

Comment:
Type inheritance isn't necessary here. You could have a single class with two constructors.